docs/Configuration.md

```diff
@@ -129,6 +129,7 @@ configuration variables
 * `SCCACHE_CONF` configuration file path
 * `SCCACHE_CACHED_CONF`
 * `SCCACHE_IDLE_TIMEOUT` how long the local daemon process waits for more client requests before exiting, in seconds. Set to `0` to run sccache permanently
+* `SCCACHE_CONNECTION_TIMEOUT` how long clients should try to connect to the server before continuing, in seconds.
```

Comment:
Overall, this idea looks good to me. However, I think it should be a config value in the `[dist]` section instead of an environment variable. Most distribution configurations aren't exposed to the environment for now.

The current name is a bit confusing from my point of view. I would expect something like `tcp_connect_timeout`.

Reply:
Should `connect_to_server` in `src/client.rs` and the functions that use it (`connect_or_start_server` in `src/command.rs`) take the config struct as an argument?